.ci/travis/travis.sh

```diff
@@ -196,6 +196,12 @@ versions)
   fi
   ;;
 
+restore-maven-cache)
+  curl -o cache.tar -Ss https://dl.dropboxusercontent.com/s/dab3q91plu962ge/checkstyle-maven-cache.tar
```

Comment:
What is the best way to whitelist url for spellchecker to fix [build](https://github.com/checkstyle/checkstyle/pull/10679/checks?check_run_id=3393335238)?

Reply:
Please replace with s3 url, steps that I've used to generate dependencies describe: https://github.com/checkstyle/checkstyle/pull/10679#discussion_r693493314

Reply:
@sukolenvo, what is size of such zip ?
I try to think how much traffic it will do on s3.
We got sponsorship on s3, but on limited amount of money per month, at least I need to warn sponsor on extra amount.

Reply:
Compressed tar is 161mb, its downloaded 5 time per build, ~700 builds a month. Its 3500 downloads or 563gb of transferred data.

Reply:
To cheat spell checker in bas we do hacks like
"https://dl.dropboxusercontent.com/s/da""b3q91pl""u962ge/checkstyle-maven-cache.tar"

dropboxusercontent we can add to whitelist file.
CI output suggests how to do this.

Reply:
https://aws.amazon.com/directconnect/pricing/

0.02 * 600gb = 12$ per month

Reply:
Here is compressed version of the same tar (30 mb smaller): https://dl.dropboxusercontent.com/s/0rw7bpmchnt0sqq/checkstyle-maven-cache.tar.gz

Reply:
dropbox url was: 
https://dl.dropboxusercontent.com/s/dab3q91plu962ge/checkstyle-maven-cache.tar

Reply:
@sukolenvo , please use 
`curl -L https://sourceforge.net/projects/checkstyle/files/drone-io/drone-io-m2-repository.tar/download`